Week3/Homework/Js-exercises/amazingNoer.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+let giveCompliment = function(name) {
```

Comment:
While this is a correct way of creating a function, it isn't how developers usually create them.
1. You can have a simply function declaration here i.e
```
function giveCompliment(name) { .. }
```
2. Or you can use `const` here instead of `let` with a new feature in JavaScript called arrow functions which you are going to learn pretty soon.
```
const giveCompliment = (name) => { ... }
```